Arrowgene.Ddon.GameServer/Chat/Command/Commands/PartyInviteCommand.cs

```diff
@@ -40,6 +41,12 @@ public override void Execute(string[] command, GameClient client, ChatMessage me
                 return;
             }
 
+            if ((client.GameMode == GameMode.BitterblackMaze) && (client.Party.MemberCount() >= 4))
```

Comment:
I don't agree with arbitrarily limiting the party invite hack. At that point, you might as well remove the command entirely as its only purpose is to bypass the normal 4 player limit

Reply:
Sam added these game configuration thing to the settings file. Should we make this configurable there?

Reply:
If we figure out how to legitimately alter party size limits it should be done in there. This command is purely a hack to bypass those limits, so if we figure out that we might as well just remove this command. If you want to for the moment have the party size limite configurable in there and use the limit in here, although im of the oppinion that this command shouldn't be limited, i wouldn't be against it

Reply:
I dropped the commit for now.